packages/react-card/src/components/Card.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { Body, Caption } from '@fluentui/react-text';
+import { Button } from '@fluentui/react-button';
+import {
+  Open16Regular,
+  ArrowReply16Regular,
+  DismissSquare20Regular,
+  MoreHorizontal16Regular,
+  MoreVertical20Regular,
+} from '@fluentui/react-icons';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { Card, CardFooter, CardHeader, CardPreview } from '../index';
+
+const useStyles = makeStyles({
+  root: {
+    background: 'white',
+    padding: '6px',
+    width: '20px',
+    height: '20px',
+
+    '> img': {
+      width: '100%',
+      height: '100%',
+    },
+  },
+});
+const LogoBackground = (props: React.HTMLAttributes<HTMLElement>) => {
+  const styles = useStyles();
+
+  return <div className={styles.root}>{props.children}</div>;
+};
+
+export const ActionCard = () => (
+  <div>
```

Comment:
nit: I don't thin the `div` is doing much here
```suggestion
  <>
```

Reply:
Good point, was there for width restricting initially but ended up removing it. Applying the suggestion.